src/App.tsx

```diff
@@ -1,17 +1,35 @@
+import { useEffect } from 'react';
 import { Route, Routes } from 'react-router-dom';
 import Hello from '~/components/Hello';
-import { DevPlayground } from './components/DevPlayground/DevPlayground';
-import { Layout } from './components/Layout/Layout';
+import useGenerator from './hooks/useGenerator';
+import { DevPlayground } from './pages/DevPlayground/DevPlayground';
+import { Layout } from './templates/Layout';
+import ThemeWrapper from './templates/ThemeWrapper';
+
+const App = () => {
+  const { generateDatabase } = useGenerator();
+
+  useEffect(() => {
```

Comment:
maybe we could install react-use library? There are many useful hooks, https://github.com/streamich/react-use
There is also `useEffectOnce`

Reply:
Ok, why not